libcpuid/cpuid_main.c

```diff
@@ -38,7 +38,7 @@
 
 /* Implementation: */
 
-static int _libcpiud_errno = ERR_OK;
+_Thread_local int _libcpiud_errno = ERR_OK;
```

Comment:
This will break compatibility with non-C11 compliant compilers. E.g. MSVC6. Probably a macro that expands to `_Thread_local` when we have C11 and `static` when not is the best way to do it.